tests/test-cases.js

```diff
@@ -733,6 +733,15 @@ var PARSE_TESTS = [
 			errors: []
 		}
 	},
+	{
+		description: "Header rows are striped out of white spaces when stripHeaders is set ",
+		input: '  A , B  ,  C  \r\na,b,c',
```

Comment:
This tests the same thing as `trimHeaders`. You should write a test that actually demonstrates the unique function of this which is to remove all white space from a header entirely. Is your intention to convert a header like `full name` to `fullname`?